manifests/c/ChilliBits/Spice/0.7.1/ChilliBits.Spice.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: ChilliBits.Spice
+PackageVersion: 0.7.1
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: msi
+Scope: machine
+InstallModes:
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/spicelang/spice/releases/download/0.7.1/spice_x64_setup.msi
+  InstallerSha256: 5b2dd41a8811d1c898624967e08a4be7700d21c0ff9b78de78f34521582995b1
+  ProductCode: "{CE9D9BA8-0EBA-4DA8-A473-7D9C44600583}"
```

```suggestion
  ProductCode: "{A70BEF44-08F4-405B-A361-349257A8C2FD}"
```
This must match Windows Registry.

---

Hi @ItzLevvie. I think the bot merged the pr accidentally. I just created a [follow-up pr](https://github.com/microsoft/winget-pkgs/pull/50338) for your suggested changes.